awx/main/models/inventory.py

```diff
@@ -1743,6 +1743,10 @@ def preferred_instance_groups(self):
 
     @property
     def ansible_virtualenv_path(self):
+        if self.inventory_source and self.inventory_source.source_project:
```

Comment:
@AlanCoding @wenottingham what do you think about this 🤔

Reply:
Looks like exactly what was asked for.

I would like @wenottingham's approval for merge. In terms of implementation, I don't see any problems.

Reply:
It does what it says it does.

This gives the user another way to both:
- customize how inventory is updated for esoteric cases
- botch how inventory is updated if they do odd things in the venv

It's fine with that known caveat.